Sources/SwiftSyntax/SyntaxRewriter.swift.gyb

```diff
@@ -98,12 +98,23 @@ open class SyntaxVisitor {
     public init() {}
 % for node in SYNTAX_NODES:
 %   if is_visitable(node):
-  open func visit(_ node: ${node.name}) {
-    visitChildren(node)
+  open func visit(_ node: ${node.name}) -> Bool {
```

Comment:
Can we add a doc comment specifying what the return value does? And: Do we ever return false here?